.github/workflows/gh-action.yml

```diff
@@ -0,0 +1,40 @@
+name: Test GitHub Action installer
+on:
+  merge_group:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - install-from-binstall-release.ps1
+      - install-from-binstall-release.sh
+      - action.yml
+  push:
+    branches:
+      - main
+    paths:
+      - install-from-binstall-release.ps1
+      - install-from-binstall-release.sh
+      - action.yml
+
+jobs:
+  test-gha-installer:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ macos-latest, ubuntu-latest, windows-latest ]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install cargo-binstall
+        uses: ./ # uses action.yml from root of the repo
+
+      - name: Verify successful installation: Display cargo-binstall's help
```

Comment:
CI says there's a syntax error here.
I gues it could be a tab/space issue.

Reply:
stoopid, should be good now